public/app/features/plugins/plugin_loader.ts

```diff
@@ -68,6 +69,7 @@ function exposeToPlugin(name: string, component: any) {
   });
 }
 
+exposeToPlugin('@grafana/data', grafanaData);
 exposeToPlugin('@grafana/ui', grafanaUI);
 exposeToPlugin('@grafana/runtime', grafanaRT);
```

Comment:
I know this ain't part of this PR, but runtime should be exposed as `grafanaRuntime`

Reply:
fixed